### DISCUSSION TITLE
How to actually use this?

### DISCUSSION THREAD

**Discussion by cakir-enes** [willothy/flatten.nvim#45] [Q&A] [answered]
Using LunarVim I have this in my config
```
{
    'willothy/flatten.nvim',
    config = true,
    -- or pass configuration with
    opts = {
      window = {
        open = "alternate"
      },
    },
    -- Ensure that it runs first to minimize delay when opening file from terminal
    lazy = false,
    priority = 1001,
  }
```

Then doing
```
> nvim somefile
:ToggleTerm
> nvim someotherfile
```
Open the file in toggle term window, playing with `window.open` didnt work either. My goal is using `broot` as file manager and opening files in host nvim (I guess after making this work I just need to set `VISUAL` and `EDITOR` variables)

**Comment by willothy** [comment, marked as answer]
Yes, $VISUAL / $EDITOR need to be set to `nvim` for this to work. Did you end up getting it working?

**Reply by cakir-enes** [reply]
Yes, turns out its just alias problems `lvim` `nvim` thanks.
